launch_ros/launch_ros/actions/node.py

```diff
@@ -235,6 +237,11 @@ def parse(cls, entity: Entity, parser: Parser):
         if args is not None:
             kwargs['arguments'] = super()._parse_cmdline(args, parser)
         node_name = entity.get_attr('node-name', optional=True)
+        if node_name is not None:
+            warnings.warn(
+                "The attribute 'node-name' is deprecated. Use the attribute 'name' instead.")
+            kwargs['node_name'] = parser.parse_substitution(node_name)
+        node_name = entity.get_attr('name', optional=True)
```

Comment:
@jacobperron FYI `name` is also an attribute for plain processes. This change means it will no longer be possible to set both independently (it currently is possible in Python). Probably not the most relevant feature, though if node name is given a suffix to make it anon, that'll show in logs from now on.

Reply:
I'd overlooked that `name` was being used for ExecuteProcess. I've proposed a solution in 61503c9. We avoid forwarding the 'name' attribute to ExecuteProcess and expose a new attribute for node, 'exec-name', that keeps the feature of setting the process node.